src/cred/index.js

```diff
@@ -68,7 +68,7 @@ export function setPhoneNumber(lock, value) {
 }
 
 export function validatePhoneNumber(phoneNumber) {
-  const regExp = /^[0-9]{1,14}$/;
+  const regExp = /^[0-9]([0-9 -])*[0-9]$/;
```

Comment:
I think it is the same, and might even be expected in this context, but the regulars expressions are not equivalent, the first one accepts "9", whereas this one requires at least 2 digits "99"